### DISCUSSION TITLE
Update Mermaid to latest version

### DISCUSSION THREAD

**Discussion by opentokix** [outline/outline#7791] [Ideas]
Mermaid version in outline:latest is 2+ years old 9.3.0 
https://github.com/outline/outline/blob/f0add849f924a706cb2a0874b4520f3ec8b1f892/package.json#L152C1-L153C1 

Time to update?

**Comment by tommoor** [comment]
Lots of other attempts and discussion at this if you have a search, it will need a concerted effort to happen as it's not just a matter of bumping the number.

**Comment by owanio1992** [comment]
our team really like mermaid new feature  
so I have rebuild outline with new mermaid version for out team use
you can use my container image, but I not guarantee stable and handle any issue   
https://hub.docker.com/repository/docker/owanio1992/outline/general  

ref. PR
https://github.com/outline/outline/pull/7865
https://github.com/outline/outline/pull/6869

**Comment by EHRETic** [comment]
Hi there 😉

Just my 5 cents... I understood it is always a struggle to update it, but I've seen some of 11.X capabilities and this would really ease IT documentation if we could use the new architecture diagrams...

https://mermaid.js.org/syntax/architecture.html

Thanks considering it sometime! 😊
